### ISSUE TITLE
Does this support Catalina?

### ISSUE THREAD

**Issue by doghouch** [iamchating/Lenovo_Ideapad_730s#1]
I've tried using this in Catalina but the laptop reboots immediately after logging in.

(Edit: yes, I know that this EFI config. is for Mojave)

**Comment by dkudrin** [comment]
@doghouch Hi, have you succeed with Catalina? Have you tried OpenCore? I worry about charging via USB-C, does it work?

**Comment by doghouch** [comment]
> @doghouch Hi, have you succeed with Catalina? Have you tried OpenCore? I worry about charging via USB-C, does it work?

I've tried OpenCore on my desktop machine but not on my laptop. I can try later -- though I'd prefer using Clover given how I like switching between Windows,  macOS & CentOS.

I'll update you if I manage to get it to work.

**Comment by iamchating** [comment]
> @doghouch Hi, have you succeed with Catalina? Have you tried OpenCore? I worry about charging via USB-C, does it work?

USB-C Charge is handled by hardware, no special setup needed

**Comment by iamchating** [comment]
I am sorry I had gave this laptop away to my nephew and I can not update this project any more

For Catalina, maybe OpenCore is better. This EFI gives some hints to bringup devices to OSX and maybe you need re-implement it by OpenCore

**Comment by doghouch** [comment]
@iamchating  

No, but I’ve had some success with Clover. I managed to get the system to boot but the old DSDT seems to break VoodooI2C. 

So far, on Catalina:

- iGPU acceleration works
- Display brightness works (from the old DSDT)
- WiFi/Bluetooth (using a DW1820/Broadcom card + BrcmFixup) works
- PS2 Keyboard works
- System audio works (speakers, USB audio, etc.)

What I’m trying to fix:

- VoodooI2C/Trackpad

I’ll push my current EFI configuration for others to see later — thanks again for the original Clover config! No worries about needing to update it :)

**Comment by seven-of-eleven** [comment]
@doghouch did you manage to get the Trackpad working? I have Big Sur installed with OpenCore but no trackpad.

**Comment by doghouch** [comment]
> @doghouch did you manage to get the Trackpad working? I have Big Sur installed with OpenCore but no trackpad.

I did manage to get trackpad working (with Clover).

I can try to find my Clover folder if you’d like — I  sold the laptop in favour of the new M1 Mac haha.

**Comment by seven-of-eleven** [comment]
@doghouch No worries. I've found another clover config that works as well. I had started with Opencore but had no trackpad. Enjoy the M1. Everything I've seen of them shows they are amazing units.

**Comment by doghouch** [comment]
> @doghouch No worries. I've found another clover config that works as well. I had started with Opencore but had no trackpad. Enjoy the M1. Everything I've seen of them shows they are amazing units.

Yeah — app compatibility is at ~95% for what I do but I still miss having an Intel Hackintosh.

Glad you found a working Clover EFI folder, though — OpenCore is notoriously hard to get working with the 730S :(
